### PR TITLE
Contact validation

### PR DESCRIPTION
**Pull request by 2004-okari**
In this project:

1. Used JavaScript to validate forms

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,3 +15,67 @@ closeUp.addEventListener('click', closing);
...
+
+  if (email.length === 0) {
+    emailError.innerHTML = 'Email is required';
+    return false;
+  }
+
+  if (!email.match(/^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|.(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/)) {
+    emailError.innerHTML = 'Invalid email';
+    return false;
+  }
```
Comment (LionRouge1) [inline index.js]: - [ ] Nice work implementing the validation, it looks really good :heart_eyes: However it requested in the project requirement that the form is submit when the email field contains only lowercase(see image). Kindly include that logic in you validation to follow the project requirement.
Screenshot from 2023-06-23 14-38-25
### Your project validation :point_down: 
Screenshot from 2023-06-23 14-29-08

**Review by LionRouge1 — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (LionRouge1) [inline index.js]: - [ ] Nice work implementing the validation, it looks really good :heart_eyes: However it requested in the project requirement that the form is submitted when the email field contains only lowercase(see image). Kindly include that logic in you validation to follow the project requirement.
Screenshot from 2023-06-23 14-38-25
### Your project validation :point_down: 
Screenshot from 2023-06-23 14-29-08

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @2004-okari, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job implementing the validation for all the field :1st_place_medal: 
- Nice work making the validation message near the particular field :+1: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @2004-okari, @NalianAchei2020,

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Form submitted only when email is in lowercase ✔️
 
## [Optional] suggestions:
* For future pull requests, please notice that your branch name `contact_validation` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-contact_validation`. This makes it easier to track the changes and the purpose of each branch.

- [ ]  Next time, and for future commits, I kindly suggest that you please write more descriptive commit messages and make use of imperative mood in the subject line for your commit messages.
:x: Bad Commit Messages :x:
image

### ✔️ Good Commit Messages: ✔️
✔️  Add fix for dark mode toggle state
✔️  Update README file.
✔️  Add styles to h1.

- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);

 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**